.vscode/settings.json

```diff
@@ -1,32 +0,0 @@
-{
-    "markdown.docsetName": "dotnet",
```

Comment:
@BillWagner is this file deletion intentional?

Reply:
Yes. It was added to our [.gitignore](https://github.com/dotnet/docs/blob/main/.gitignore#L30) recently.